### PR TITLE
implement the display trait

### PR DESCRIPTION
**Pull request by PascaleBorscia**
I implement the Display trait for the FlowId struct.

### CODE REVIEW

File: src/flow_id.rs [hunk 1]
```diff
@@ -3,8 +3,8 @@ use std::hash::{Hash, Hasher};
 use std::net::{IpAddr, Ipv4Addr};
```
Comment (tigroo31) [inline src/flow_id.rs]: I would prefer a git commit --amend into the previous commit, to keep a clear git history :)

File: src/flow_id.rs [hunk 2]
```diff
@@ -3,6 +3,7 @@ use std::hash::{Hash, Hasher};
...
 use std::str::FromStr;
 
+use core::fmt;
```
Comment (tigroo31) [inline src/flow_id.rs]: use std::fmt; instead no?

File: src/flow_id.rs [hunk 3]
```diff
@@ -206,4 +217,10 @@ mod tests {
         flow2.hash(&mut hasher2);
         assert_eq!(hasher1.finish(), hasher2.finish());
     }
+
+    #[test]
+    fn test_display() {
+        let flow = build_local_flow_id();
+        assert_eq!(format!("{}", flow), "127.0.0.1-192.168.0.1-8001-8002-17")
```
Comment (tigroo31) [inline src/flow_id.rs]: Yes.
We may use into our code the method to_string() dedicated to variable (format is more log oriented:

        assert_eq!(flow.to_string(), "127.0.0.1-192.168.0.1-8001-8002-17");

**Review by tigroo31 — CHANGES_REQUESTED** [review]
Code ok.
Tests ok.

File: src/flow_id.rs (hunk 3 above)
Comment (tigroo31) [reply src/flow_id.rs]: to trigger the deployments (Crates and Docs), you must increase the version into Cargo.toml:

[package]
name = "libflow"
version = "0.1.4"

**Review by tigroo31 — CHANGES_REQUESTED** [review]
Adding the new version 1.3.4 declaration request.

File: src/flow_id.rs (hunk 1 above)
Comment (PascaleBorscia) [reply src/flow_id.rs]: 👍

File: src/flow_id.rs (hunk 2 above)
Comment (PascaleBorscia) [reply src/flow_id.rs]: 👍

File: src/flow_id.rs (hunk 3 above)
Comment (PascaleBorscia) [reply src/flow_id.rs]: 👍

**Review by tigroo31 — APPROVED** [review]

**Pull request merged**